src/tests/bdd/create_lookup.feature

```diff
@@ -1,19 +1,8 @@
 Feature: Create a lookup table
+    Background: There are data sources in the system
+    Given the system data source and SIMOS core package are available
 
-  Scenario: Create a lookup table
-    Given i access the resource url "/api/v1/lookup/my-new-lookup"
+  Scenario: System admins want to create a recipe lookup for the DMSS - SIMOS/recipe_links folder
+    Given i access the resource url "/api/v1/lookup/system/SIMOS/recipe_links?application=dmss"
```

Comment:
Just a suggestion, change endpoint to: `/api/v1/lapplications/dmss/lookup/system/SIMOS/recipe_links`

Reply:
how about `/api/v1/applications/dmss?recipe_package=system/SIMOS/recipe_links`?

The resource is the DMSS application, recipe package is just a query param, and should be kept out of the url